### PR TITLE
fix(op_crates/web): handler order when reassign

### PR DESCRIPTION
**Pull request by benjamingr**
This fixes the nasty bug described in https://github.com/denoland/deno/pull/8230#issuecomment-722945098 

cc @kitsonk

### CODE REVIEW

**Comment by benjamingr** [comment]
cc @kitsonk @kt3k any chance this can get a review/merge? I have follow-up work building on this I want to start working on ^^

File: op_crates/web/02_abort_signal.js [hunk 1]
```diff
@@ -64,22 +64,35 @@
   }
 
   const handlerSymbol = Symbol("eventHandlers");
+
+  function makeWrappedHandler(handler) {
+    function wrappedHandler(...args) {
+      if (typeof wrappedHandler.handler !== "function") {
+        return;
+      }
```
Comment (bartlomieju) [inline op_crates/web/02_abort_signal.js:72]: In what situation this branch would be true?

Comment (benjamingr) [reply op_crates/web/02_abort_signal.js:72]: In the test basically:
```js
ee.onabort = () => console.log('irrelevant');
ee.onabort = () => null;
ee.addEventListener('abort', e => console.log(1);
ee.onabort = () => console.log(2);
```

In this case, the target has to fire "1 2" and not "2 1" so we need to "keep the slot".

the DOM is weird like that 🤷

Comment (benjamingr) [reply op_crates/web/02_abort_signal.js:72]: The test tests this btw

Comment (bartlomieju) [reply op_crates/web/02_abort_signal.js:72]: Thanks, that makes sense 👍

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @benjamingr!

**Comment by benjamingr** [comment]
Great thanks, I'll keep em coming - I'd like to make this fix to other EventTargets that have handlers (like sockets) but I am unsure how to share code between web crates. 

Should I just copy-paste those fixes to those files and open a PR until you figure out how you want sharing code between them to look?

**Comment by bartlomieju** [comment]
@benjamingr I think a little copy-paste now is fine - indeed we don't yet have a good way to share this functionality between crates.

**Comment by benjamingr** [comment]
Ok, thanks!

**Pull request merged**